tools/common/Driver.cs

```diff
@@ -47,17 +52,69 @@ static bool ParseOptions (Application app, Mono.Options.OptionSet options, strin
 			Action a = Action.None; // Need a temporary local variable, since anonymous functions can't write directly to ref/out arguments.
 
 			options.Add ("h|?|help", "Displays the help", v => a = Action.Help);
+			options.Add ("f|force", "Forces the recompilation of code, regardless of timestamps", v => Force = true);
+			options.Add ("cache=", "Specify the directory where temporary build files will be cached", v => app.Cache.Location = v);
 			options.Add ("version", "Output version information and exit.", v => a = Action.Version);
 			options.Add ("v|verbose", "Specify how verbose the output should be. This can be passed multiple times to increase the verbosity.", v => Verbosity++);
 			options.Add ("q|quiet", "Specify how quiet the output should be. This can be passed multiple times to increase the silence.", v => Verbosity--);
+			options.Add ("debug:", "Build a debug app. If AOT-compiling, will also generate native debug code for the specified assembly (set to 'all' to generate debug code for all assemblies, the default is to generate debug code for user assemblies only)", v => {
```

Comment:
nit: the last sentence needs a `.` at the end.

Reply:
@stephen-hawley fixed